cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/constants.js

```diff
@@ -5,3 +5,5 @@
 
 export const DATASOURCE_NAME = 'query-cluster';
 export const WORKSPACE_NAME = 'query-workspace';
+export const START_TIME = 'Dec 31, 2020 @ 00:00:00.000';
+export const END_TIME = 'Dec 31, 2022 @ 23:59:59.999';
```

Comment:
could we relax these dates to not make it so close to our dataset? more convo here: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/9047#discussion_r1890745272

Reply:
good point. let me update it with other comments.

Reply:
fixed it